website/docs/reference/global-configs/command-line-flags.md

```diff
@@ -4,60 +4,91 @@ id: "command-line-flags"
 sidebar: "Command line flags"
 ---
 
-Command line (CLI) flags immediately follow `dbt` and precede your subcommand. When set, CLI flags override environment variables and profile configs.
+For consistency, command line (CLI) flags should come right after the `dbt` prefix and its subcommands. This includes "global" flags (supported for all commands). When set, CLI flags override environment variables and profile configs. 
```

Comment:
Just a heads up that CLI technically stands for "command-line interface."
```suggestion
For consistency, command-line interface (CLI) flags should come right after the `dbt` prefix and its subcommands. This includes "global" flags (supported for all commands). When set, CLI flags override environment variables and profile configs. 
```

Reply:
oh thank you! i keep thinking command line is just cli 🙈